frontend/js/components/map.js

```diff
@@ -139,15 +138,11 @@ export class LeafletMap extends React.Component {
             return <IssueMarker {...props} />
         })
 
-        let extents = geojsonExtent(JSON.parse(JSON.stringify(geojson)));
```

Comment:
geoJsonExtent removes the 'id' attribute from each object in features, breaking our stuff ...